website/docs/rbi.md

```diff
@@ -265,6 +265,10 @@ and predictable over time.
 So if/when we wish to upgrade an RBI that we've already checked in to our
 project's version control, we can do so explicitly with the `srb rbi` command.
 
+## Generating RBIs from YARD
+
+There is community-maintained project which allows you to generate Sorbet RBI files from YARD docs: https://github.com/AaronC81/sord.
+
```

Comment:
Pretty much every project under the Projects section in the community doc is related to RBIs right now (sorbet-typed, sorbet-rails, and Sord are all for generating or sharing RBIs).

What do you think about just linking to the Community section here? I don't want to have to duplicate everything from the community section into rbi.md; I'd like to be able to keep this doc specific to Sorbet core if possible.

If searching for the term "Sord" is what you're looking for, I'd welcome a change that makes it so that the community page shows up in the index results.